docs/platform/howto/make-super-admin.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: Make users super admin
+title: Super admin
 ---
 
 import ActionsIcon from "@site/static/images/icons/more.svg";
 
-Super admin have full access to an organization and its settings as well
-as all of its organizational units, projects, and services.
+The super admin role has full access to an organization, its billing and settings, and all its organizational units, projects, and services.
 
-## Make a user a super admin
```

Comment:
I would suggest keep this section title.

Reply:
Per Slack discussion, adding related pages for now to include the heading for SEO.